packages/svelte-ux/src/routes/+layout.svelte

```diff
@@ -26,13 +26,26 @@
   $: title = data.pr_id ? `🚧 (pr:${data.pr_id}) - ${baseTitle}` : baseTitle;
 
   settings({
+    // Usefull to test different locales with the docs
```

Comment:
See comment about adding global settings dialog (in future).  Will help a lot with this